### DISCUSSION TITLE
GET Shipment ID (ARN) for PO#

### DISCUSSION THREAD

**Discussion by gmamacos** [amzn/selling-partner-api-models#576] [Ideas]
For suppliers using EDI, the Shipment ID or ARN (also known as the routing request or dispatch# or load#) is required as a mandatory field on the ASN (EDI 856 document). This is the number that Amazon provides for the booking of the load pickup.
		
Currently, the only way to acquire this Shipment ID/ARN is via the Vendor Central UI, where one is able to export / download data as a csv file.
		
This is a manual bottleneck for all fully integrated EDI providers using 850, 856 & 810 EDI docs.
		
Please support a GET call where a PO# is provided and the Shipment ID / ARN is returned.
		
A screenshot is attached where both pertinent fields are each highlighted by a red rectangle.
		
Thank you.  

![Shipment ID ARN in VendorCentral](https://user-images.githubusercontent.com/10473759/131200102-c38cdede-f356-4997-8c91-218f43a6242c.jpg)

**Comment by adam12104** [comment]
Has anybody found a solution to this as of yet?

**Reply by jenkweb** [reply]
Also looking for solutions regarding shipment ARN requirement. Populating this number on EDI 856 consistently with no access to the ARN data on the Amazon side (outside the website UI) is problematic and an ongoing issue.

**Reply by rakeshchaubey** [reply]
Has anybody found any solutions for this ? We are also having this issue !

**Reply by alexartisans** [reply]
no way to do that in v0 Inbound API. New one WILL have this functionality (but not yet)

**Reply by apawar-hub** [reply]
its been over a year, any idea where these has been taken care? I don't see any further update on this thread
